packages/edit-navigation/docs/user-documentation.md

```diff
@@ -0,0 +1,72 @@
+Note: this documentation is a work in progress.
+
+# Block-based menu editor
+
+The Block-based Navigation Editor brings the power of blocks to the Appearance > Navigation section in the WordPress Administration Screens allowing you use blocks to create menus.
+
+## How to use the editor
+
+The interface replicates the Post Editor experience, allowing you to use similar workflows like drag and drop.
+
+The editor provides access to all the menus created using the previous Menu screen in WordPress.
```

Comment:
```suggestion
The editor provides access to all/any of the menus you created using the classic `Menus` screen in WordPress.
```

Reply:
It's a nitpick, but I don't think 'you' is correct because different users could've created the menus, not just the reader. 😄 

I'm a bit hesitant about saying 'classic' right now. Widget docs use the term 'previous', so I think it's good to stay consistent. This might change if we decide to use the term 'classic' in the UI or as a plugin.

The sentence is a bit awkward, but I don't know, I think I'll leave it for now.